app/views/shared/_adventure_form.html.erb

```diff
@@ -62,9 +76,20 @@
 <% end %>
 
 <script>
-  window.onload = togglePasswordField(document.getElementById("adventure_has_password"))
+  window.onload = togglePasswordField(document.getElementById("adventure_has_password"), document.getElementById("adventure_has_age_limit"))
```

Comment:
looks like you're passing two dom elements to the `togglePasswordField`, which doesn't seem quite right. Should this make use of the new `toggles` function below?